portfolio/src/main/webapp/script.js

```diff
@@ -40,27 +40,37 @@ function highlightTab() {
 }
 
 function fetchComments() {
-  (fetch('/comments').then(response => response.json()).then((comments) => {
+  // get num of comments to query
+  const selectBar = document.getElementById("numComments");
+  const maxComments = selectBar.options[selectBar.selectedIndex].value;
+
+  (fetch('/comments?numComments=' + maxComments).then(response => response.json()).then((comments) => {
```

Comment:
Is there a reason you are wrapping fetch in parentheses? It seems like it should not be needed.